### ISSUE TITLE
Mocking Variable Length Array (VLA) not working

### ISSUE THREAD

**Issue by parmi93** [ThrowTheSwitch/Ceedling#892]
```c
void read(size_t buff_size, char (*buffer)[buff_size]);
```
The auto-generated code for the mocking of this type of functions does not work.

![image](https://github.com/ThrowTheSwitch/Ceedling/assets/61920513/9ae3ed7e-3ac4-4a95-a269-eada4e8716db)

**Comment by parmi93** [comment]
Is there any plan to support that?

**Comment by mvandervoord** [comment]
Yes. Arrays and pointers are getting a big overhaul in the next major release. Both are insufficient as they only handle the most common cases.
